### PR TITLE
[WIP][FIX] Flake8 issues

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
Travis is green :smile:

**Comment by pedrobaeza** [comment]
:+1: Fast-tracking then

File: mis_builder/static/src/xml/mis_widget.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
                                     </a>
                                 </t>
                                 <t t-if="!value_value.drilldown">
-                                    <t t-esc="value_value.val_r"/>
+                                    <t t-esc="value_value.val_r" t-field-options='{"widget": "monetary"}'/>
```
Comment (yvaucher) [inline mis_builder/static/src/xml/mis_widget.xml:45]: This doesn't seems linked to pep8. Was it intended ?

**Comment by yvaucher** [comment]
@pedrobaeza Wrong fast tracking

File: mis_builder/static/src/xml/mis_widget.xml (hunk 1 above)
Comment (pedrobaeza) [inline mis_builder/static/src/xml/mis_widget.xml:45]: Sorry. I missed this part in my review before fast-tracking, but I think it doesn't hurt, and most as they're the authors of the module.

**Comment by pedrobaeza** [comment]
Yeah, I see. We can revert if you consider so.

**Comment by adrienpeiffer** [comment]
@yvaucher @pedrobaeza I will correct it

**Comment by yvaucher** [comment]
I guess this change was included in https://github.com/OCA/account-financial-reporting/pull/151

**Pull request merged**